analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -164,11 +164,12 @@ def construct_analyzer_cmd(self, result_handler):
                             # Turn off clang hardcoded checkers list.
                             '--analyzer-no-default-checks']
 
-            for plugin in config.analyzer_plugins:
-                analyzer_cmd.extend(["-Xclang", "-plugin",
-                                     "-Xclang", "checkercfg",
-                                     "-Xclang", "-load",
-                                     "-Xclang", plugin])
+            if env.is_analyzer_from_path():
```

Comment:
Please add a comment, that do not load the plugin because it might be incompatible (not build for the configured clang binary)